index.html

```diff
@@ -1,26 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
 	<head>
+		<title>Homework Assignment Three</title>
     	<meta charset="utf-8">
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
 		<!-- external CSS link -->
 		<link rel="stylesheet" href="css/reset.css">
 		<link rel="stylesheet" href="css/style.css">
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
 
 	</head>
 	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
+		<div class="topbar clearfix">
+		<br>
+			<div class="leftfloat">Relaxr</div>
+			<div class="middlefloat"></div>
+			<div class="rightfloat alignright">About FAQ Team Contact Us Blog</div>
 		</div><!-- #container -->
+		<br>
+		<br>
+		<section id="container" class="clearfix">
+			<div>
+				<div class="main">
+					<header>How I implemented Relaxr in 2 weeks and changed my life</header>
+					<img src="blog_photo1.jpg">
+					<p><span class="dropcap">L</span>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+					<br>
```

Comment:
Rather than using lots of `<br>` elements, I recommend using `padding` or `margin` to create white space.